### PR TITLE
Fix a typo in FlinkRunner.toString

### PR DESCRIPTION
**Pull request by peihe**
R: @tgroh @kennknowles

### CODE REVIEW

**Comment by tgroh** [comment]
LGTM

**Comment by kennknowles** [comment]
Even though this is pretty trivial, still send Flink-runner related PRs to the folks who manage it, @mxm and @aljoscha.

Even for this PR, this `toString()` is copied from the `DataflowRunner`, and maybe they want to do something entirely different.

File: runners/flink/runner/src/main/java/org/apache/beam/runners/flink/FlinkRunner.java [hunk 1]
```diff
@@ -144,7 +144,7 @@ public FlinkPipelineOptions getPipelineOptions() {
 
   @Override
   public String toString() {
-    return "DataflowRunner#" + hashCode();
+    return "FlinkRunner#" + hashCode();
```
Comment (mxm) [inline runners/flink/runner/src/main/java/org/apache/beam/runners/flink/FlinkRunner.java]: We could also include the provided PipelineOptions here. Doesn't have to be part of this PR though.

**Comment by mxm** [comment]
LGTM

**Pull request closed without merging**